WeatherApp/src/main/java/org/example/Location.java

```diff
@@ -0,0 +1,41 @@
+package main.java.org.example;
+
+public class Location {
```

Comment:
1. To miało implementować/dziedziczyć po ApiQuery.
2. "Location" jest naturalną nazwą dla kilku różnych rzeczy w naszym programie, dlatego proponuję jednak CityQuery (bo ta klasa służy do tworzenia query wyszukującego lokalizacje po miastach).
3. Obiekty mają być niemutowalne.
Do poprawy.

Reply:
ok, poprawiam